src/web/routes/cookies.js

```diff
@@ -0,0 +1,16 @@
+const pageContent = {
+  // TODO setup cookies page to handle translations HTBHF-586
+  cookies: 'Cookies'
```

Comment:
Should cookies be set as the heading rather than cookies?
```
heading: 'Cookies'
```
.. also, is it a big deal to setup translations for this? otherwise we are merging non translatable content into the master branch, is this OK (maybe its fine)?

Reply:
Had a word with @YetAnotherMatt and decided that no translation would be ok for now.

Reply:
Updated to heading